install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -994,12 +994,14 @@ hubble:
       # false, the certs for Hubble server need to be provided by setting
       # appropriate values below.
       enabled: true
-      # -- Set the method to auto-generate certificates. Supported values:
+      # -- Sets the method to auto-generate certificates. Supported values:
+      # - cronJob:      This method uses a Kubernetes CronJob to generate any
```

Comment:
nit: "cronjob" would match capitalization of "certmanager" and reads better

Reply:
make sense, we could update the value here and do a case-insensitive check in the template for backward compatibility.

Reply:
Thinking back about this we have `.Values.certgen.annotations.cronJob`, so we'll keep cronJob for now for consistency with this annotation Helm value.